src/main/java/org/apereo/openlrs/storage/mongo/NormalizedMongoTierTwoStorage.java

```diff
@@ -89,13 +96,15 @@ public OpenLRSEntity save(OpenLRSEntity entity) {
     }
 
     @Override
+    @Query
     public List<OpenLRSEntity> findWithFilters(Map<String, String> filters) {
-        throw new UnsupportedOperationException("OpenLRS searching by filters not yet implemented for MongoDB");
+        Sort sortSpec = new Sort(Sort.Direction.DESC, "id"); // Most recent events first
```

Comment:
This was recommended by some MongoDB documentation.  It only works because the IDs it assigns are always increasing.  This technique won't work with all DBs, though.

On the other hand, I wonder if the ID counter could ever get reset to a lower value.  I'm considering adding a timestamp field and sorting on that instead.